client/src/components/StoreCard/StoreCard.tsx

```diff
@@ -36,7 +39,8 @@ function StoreCard({
         alignItems="center"
       >
         <Grid item xs={4}>
-          Media
+          {store.media && <img src={store.media} height={MAX_CARD_HEIGHT} />}
+          {!store.media && <p>Media Placeholder</p>}
```

Comment:
nit: Instead of doing this here, i think would be cleaner to make a smaller component that takes in an image and renders the media if present else show placeholder

Reply:
gotcha, will address this in the next PR! #122